src/main/resources/config/application.yml

```diff
@@ -20,6 +19,8 @@ server:
     mime-types: application/json,application/xml,text/html,text/xml,text/plain
 
 revocation:
+  authorization:
+    reload: 600000
```

Comment:
either seconds or even better use some Java duration format (like @vroldanbet already did in Plan B Provider) --- it should somehow be consistent with what we already have :smirk: